src/transformers/generation/logits_process.py

```diff
@@ -392,16 +392,55 @@ def __call__(self, input_ids: torch.LongTensor, scores: torch.FloatTensor) -> to
 
 class TypicalLogitsWarper(LogitsWarper):
     r"""
-    [`LogitsWarper`] that performs typical decoding. See [Typical Decoding for Natural Language
-    Generation](https://arxiv.org/abs/2202.00666) for more information.
+    [`LogitsWarper`] This warping technique aims to produce more coherent and fluent text by selecting words that have information content close to the expected information content,
+    i.e., the conditional entropy of the model. See [Typical Decoding for Natural Language Generation](https://arxiv.org/abs/2202.00666) for more information.
 
+     <Tip>
+      
+      When using probabilistic language models for text generation, pass do_sample=True to enable locally typical sampling with reduced repetitions and improved coherence.
+    
+    </Tip>
+    
     Args:
         mass (`float`):
-            Value of typical_p between 0 and 1 inclusive, defaults to 0.9.
+             The proportion of probability mass to retain while warping the logits. The value should be between 0 and 1.
+                Higher values (close to 1.0) retain more probability mass, leading to more typical sampling, whereas lower
+                values (close to 0.0) retain less probability mass, leading to more diverse sampling. The default is 0.9.
         filter_value (`float`, *optional*, defaults to `-float("Inf")`):
-            All filtered values will be set to this float value.
+             The value used to filter out logits that fall below this threshold. Any logits less than this value will be
+                set to -infinity before applying the softmax function. This helps in excluding unlikely tokens during sampling.
+                Default is -infinity.
```

Comment:
nit: same as above re indent. We don't need to say the default in the type info as `, defaults to ...` above 

```suggestion
             The value used to filter out logits that fall below this threshold. Any logits less than this value will be
             set to -infinity before applying the softmax function. This helps in excluding unlikely tokens during sampling.
```